tests/dtm/test_rgw_s3_process_restart.py

```diff
@@ -0,0 +1,287 @@
+        self.access_key = resp[1]["access_key"]
+        self.secret_key = resp[1]["secret_key"]
+        self.iam_user = {'s3_acc': {'accesskey': self.access_key, 'secretkey': self.secret_key,
+                                    'user_name': self.s3acc_name}}
+        self.s3_test_obj = S3TestLib(access_key=self.access_key, secret_key=self.secret_key,
+                                     endpoint_url=S3_CFG["s3_url"])
+        self.dtm_obj = DTMRecoveryTestLib(access_key=self.access_key, secret_key=self.secret_key,
+                                          retry=5)
+        self.log.info("Created IAM user with name %s", self.s3acc_name)
+        if not os.path.exists(self.test_dir_path):
+            system_utils.make_dirs(self.test_dir_path)
+
+    def teardown_method(self):
+        """Teardown class method."""
+        if not self.test_completed:
+            self.log.info("Test Failure observed, collecting support bundle")
+            path = os.path.join(LOG_DIR, LATEST_LOG_FOLDER)
+            resp = support_bundle_utils.collect_support_bundle_k8s(
+                local_dir_path=path, scripts_path=const.K8S_SCRIPTS_PATH)
+            assert_utils.assert_true(resp)
+        if self.iam_user:
+            self.log.info("Cleanup: Cleaning created IAM users and buckets.")
+            resp = self.ha_obj.delete_s3_acc_buckets_objects(self.iam_user)
+            assert_utils.assert_true(resp[0], resp[1])
+        if os.path.exists(self.test_dir_path):
+            system_utils.remove_dirs(self.test_dir_path)
+
+    @pytest.mark.lc
+    @pytest.mark.dtm
+    @pytest.mark.tags("TEST-42244")
+    def test_read_during_rgw_svc_restart(self):
+        """Verify READs during rgw_s3 service restart using pkill."""
+        self.log.info("STARTED: Verify READs during rgw_s3 service restart using pkill")
+        log_file_prefix = 'test-42244'
+        que = multiprocessing.Queue()
+
+        self.log.info("Step 1: Perform write operation for background read")
+        self.dtm_obj.perform_write_op(bucket_prefix=self.bucket_name,
+                                      object_prefix=self.object_name,
+                                      no_of_clients=self.test_cfg['clients'],
+                                      no_of_samples=self.test_cfg['samples'],
+                                      obj_size=self.test_cfg['size'],
+                                      log_file_prefix=log_file_prefix, queue=que)
+        resp = que.get()
+        assert_utils.assert_true(resp[0], resp[1])
+        workload_info = resp[1]
+        self.log.info("Step 2: Perform Read Operations on the data written in step 1 in background")
+        proc_read_op = multiprocessing.Process(target=self.dtm_obj.perform_ops,
+                                               args=(workload_info, que, False, True, True))
+        proc_read_op.start()
+
+        self.log.info("Step 3: Perform rgw_s3 Process Restart for %s times During Read "
```

Comment:
Here we need to restart rgw only once,
TEST-42247 can be used to restart rgw multiple times